### DISCUSSION TITLE
Not working on Windows 11 22621.2428

### DISCUSSION THREAD

**Discussion by Stoutscientist** [valinet/ExplorerPatcher#2310] [Q&A]
I just installed this and explorer disappeared. I restarted my PC and I got this error:

![image](https://github.com/valinet/ExplorerPatcher/assets/90054049/bceb9f42-bbaf-4fba-9334-25e50e994cc3)

**Comment by Stoutscientist** [comment]
I'm 99% sure I have latest version

**Comment by pyrates999** [comment]
Please download this version then as it is the latest release as of today: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2361.58.4_b157aba

**Reply by Stoutscientist** [reply]
That's what I'm using

**Comment by pyrates999** [comment]
Do you have any other software installed that modifies the windows UI like windhawk?

And try the following:

1. uninstall explorer patcher
2. delete the registry keys: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher" "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher"
3. reboot
4. install the latest explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by Stoutscientist** [reply]
I have done all of that and still doesn't work. I'm only using TranslucentTB and even tried exiting it but nothing.

**Comment by Amrsatrio** [comment]
Use the latest, and by latest I mean latest prerelease, the really latest one. It works on my TranslucentTB-installed 22621.2428 machine perfectly.

https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2361.58.4_a57471f

<img width="1355" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/5071cfd0-5f27-4c11-b276-b4982df8f9f8">

**Reply by Stoutscientist** [reply]
What the heck?? Even though this is a newer version than I'm using (came out yesterday) it still doesn't work....... I think I'll reinstall Windows and do other testing since my taskbar has been buggy anyway

**Comment by Stoutscientist** [comment]
Works great after wiping Windows, thanks y'all!
